cmd/translate/main_grpc_integration_test.go

```diff
@@ -100,16 +100,16 @@ func Test_UploadTranslationFile_gRPC(t *testing.T) {
 
 	// Requests
 
-	happyRequest := randUploadRequest(t, service.Id)
+	happyRequest := randUploadRequest(t, service.GetId())
```

Comment:
was that some new linter issue or scope creep ? 🤔

Reply:
It was a new linter issue (protogetter)